---
title: Research
nav:
  order: 1
  tooltip: Published works
---

# <img src="heart_icon.png" alt="Research" style="width: 45px; height: 50px; margin-right: 10px; vertical-align: middle;">Research

<div style="text-align: center;">  
  
   <p>I specialize in designing and implementing tractable experiments to test hypotheses generated from personal and population genetics observations. I am particularly interested in genetic heart disease and the variable expressivity that characterizes these conditions. Key to this work is the discovery, validation, and mechanistic mapping of high and low impact genetic variants. This follows my prior work on the spectrum of disease attributed to defective cilia and my reverse genetics approaches in understanding cardiac biology. I believe that understanding and integrating genetic variation within the context of local and background effects is the next phase of human genetics research. This topic is acutely relevant as clinical genetics becomes more routine, especially in diverse populations and the embrace of trait polygenicity. My current work on trying to better predict sudden cardiac death is my attempt to understand those interactions. In my view, a future with better predictive models, backed by refined molecular pathomechanisms, will revolutionize the prevention and treatment of cardiomyopathies.  My role can be divided into three main areas of investigation:
</p>

  
</div>


# <img src="heart_icon.png" alt="Research" style="width: 45px; height: 50px; margin-right: 10px; vertical-align: middle;">Approach

<div style="text-align: left;">
  <ul>
    <li><strong>Statistical inference to identify suspicious candidate variants.</strong> Rationally designed drugs with a strong basis in human genetics have repeatedly been proven to be more likely to gain FDA approval. Therefore, I focus my attention on discovering novel genetic components of disease. My approach is to identify highly penetrant variants passed down within families, and separately, look for inflation of common variants in more common diseases. I am keenly interested in how rare and common genetic variants interact to modify disease presentation, and how cumulative genetic burden amounts to a continuum of causality.</li>
  </ul>

 
 <ul>
    <li><strong>Testing the pathogenicity of those candidates.</strong> Of course, statistical inference is not enough to infer pathogenicity of novel loci. The early days of human genetics were fraught with imperfect assumptions about causality and genetic misattributions. Therefore, a critical component of my work is validation of candidate alleles. Usually, this means using animal models, or more recently, human-engineered tissues. All models are imperfect, so each decision is based on careful consideration of the appropriateness of the validation system. My preferred approach is to use orthogonal models and assay for convergent lines of evidence.</li>
 </ul>
 
 <ul>
 <li><strong>Using that knowledge to more clearly understand pathomechanisms.</strong> Once we identify a candidate variant and test whether or not it is pathogenic, the next step is to understand why. Even pathogenic variants within the same gene can have divergent molecular consequences. Pathogenicity within one gene can range from loss-of-function to overexuberance, or even deviant neofunctionalization. Therefore, in order to understand the genesis and course of disease, it is critical to understand the proximal molecular consequences stemming from mutation.</li>
</ul>
  
# <img src="heart_icon.png" alt="Research" style="width: 45px; height: 50px; margin-right: 10px; vertical-align: middle;">Current Projects

<div>

<div style="text-align: left;">
  <ul>
    <li><strong>Statistical inference to identify suspicious candidate variants.</strong> Rationally designed drugs with a strong basis in human genetics have repeatedly been proven to be more likely to gain FDA approval. Therefore, I focus my attention on discovering novel genetic components of disease. My approach is to identify highly penetrant variants passed down within families, and separately, look for inflation of common variants in more common diseases. I am keenly interested in how rare and common genetic variants interact to modify disease presentation, and how cumulative genetic burden amounts to a continuum of causality.</li>
  </ul>

 
 <ul>
    <li><strong>Testing the pathogenicity of those candidates.</strong> Of course, statistical inference is not enough to infer pathogenicity of novel loci. The early days of human genetics were fraught with imperfect assumptions about causality and genetic misattributions. Therefore, a critical component of my work is validation of candidate alleles. Usually, this means using animal models, or more recently, human-engineered tissues. All models are imperfect, so each decision is based on careful consideration of the appropriateness of the validation system. My preferred approach is to use orthogonal models and assay for convergent lines of evidence.</li>
 </ul>
 
 <ul>
 <li><strong>Using that knowledge to more clearly understand pathomechanisms.</strong> Once we identify a candidate variant and test whether or not it is pathogenic, the next step is to understand why. Even pathogenic variants within the same gene can have divergent molecular consequences. Pathogenicity within one gene can range from loss-of-function to overexuberance, or even deviant neofunctionalization. Therefore, in order to understand the genesis and course of disease, it is critical to understand the proximal molecular consequences stemming from mutation.</li>
<ul>


{% include section.html %}

{% include search-box.html %}

{% include search-info.html %}
