---
title: Research
nav:
  order: 1
  tooltip: Published works
---

# <img src="heart_icon.png" alt="Research" style="width: 45px; height: 50px; margin-right: 10px; vertical-align: middle;">Research

<div style="text-align: center;">  
  
<p align="left">
     I specialize in designing and implementing experiments to test hypotheses generated from personal and population genetics observations. I am particularly interested in genetic heart diseases and the variable expressivity that characterizes these conditions.
 </p>
<p align="left">
  The details of how cells and tissues self-organize with remarkable fidelity have become increasingly clear, yet we cannot engineer organs de novo. Simultaneously, we have made incredible advances in defining the pieces of the larger puzzle that is complex inheritence, but we remain woefully inadequate at predicting outcomes from genotypes alone. The crux of these issues is our nascent understanding of cis and trans genetic interactions. My work is in pursuit of a better understanding of how cells integrate gene expression and gene function across space and time. <strong>I am driven by a central ethos that understanding how cells interpret DNA to form healthy humans will lead to innovation that improves the quality of human life.</strong>
</p>
<p align="left">
     This work is framed around the discovery, validation, and mechanistic mapping of high and low impact genetic variants. An important extension to that approach being the interpretation of variants within the context of local modifiers, non-additive interactions, and background effects. This topic is acutely relevant as clinical genetics becomes more routine, especially in diverse populations and given the nature of trait polygenicity. In my view, multi-allelic predictive models informed by defined molecular pathomechanisms will revolutionize the prevention and treatment of cardiomyopathies. My general strategy sorts into three types of projects:
</p>

  
</div>


# <img src="heart_icon.png" alt="Research" style="width: 45px; height: 50px; margin-right: 10px; vertical-align: middle;">Approach

<div style="display: flex; align-items: start;">

  <div style="flex: 1;">
    <ul>
      <li><strong>Statistical inference to identify suspicious candidate variants.</strong> Rationally designed drugs with a strong basis in human genetics have repeatedly been proven to be more likely to gain FDA approval. Therefore, I focus my attention on discovering novel genetic components of disease. My approach is to identify highly penetrant variants passed down within families, and separately, look for inflation of common variants in more common diseases. I am keenly interested in how rare and common genetic variants interact to modify disease presentation, and how cumulative genetic burden amounts to a continuum of causality.</li>
    </ul>
  </div>
  
  <div>
    <img src="identify.png" alt="Research" style="width: 230px; height: 300px; margin-left: 20px;">
  </div>

</div>

<div style="display: flex; align-items: start;">
  
  <div>
    <img src="website.png" alt="Research" style="width: 400px; height: 400px; margin-right: 20px;">
  </div>

  <div style="flex: 1;">
    <ul>
     <li><strong>Testing the pathogenicity of those candidates.</strong> Of course, statistical inference is not enough to infer pathogenicity of novel loci. The early days of human genetics were fraught with imperfect assumptions and genetic misattributions. Therefore, a critical component of my work is validation of candidate alleles. Usually, this means using animal models, or more recently, engineered human tissues. All models are imperfect, so each decision is based on careful consideration of the appropriateness of the validation system. My preferred approach is to use orthogonal models to identify convergent lines of evidence.</li>
    </ul>
  </div>
  
 
</div>

  <ul>
 <li><strong>Using that knowledge to more clearly understand pathomechanisms.</strong> Once we identify a candidate variant and test whether or not it is pathogenic, the next step is to understand why. Even pathogenic variants within the same gene can have divergent molecular consequences, ranging from antimorphic to hypermorphic, or even neomorphic. Therefore, in order to understand the genesis and course of disease, it is critical to understand the proximal molecular consequences stemming from mutation.</li>
</ul>
  



{% include section.html %}

{% include search-box.html %}

{% include search-info.html %}
